app/build.gradle.kts

```diff
@@ -56,24 +63,18 @@ addPublication("Apk") {
     }
 
     android.applicationVariants.all {
+        val isMainVariant = name == mainVariant && false // TODO
```

Comment:
TODO ?

Reply:
Ah, good catch. I was checking if one artifact without classifier was required or not. Answer: all artifacts in the test app have a classifier: debug, staging, release, source, javadoc